### PR TITLE
docs(en-US): add next sdk integration guide

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

add next SDK integration guide

### CODE REVIEW

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+import Tabs from '@theme/Tabs';
+
+import redirectUriFigure from './assets/next-redirect-uri.png';
+import AppNote from './fragments/_app-note.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
+import AssumingUrl from './fragments/_next-assuming-url.md';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+
+# React: Integrate `@logto/react`
```
Comment (demonzoo) [inline docs/docs/recipes/integrate-logto/next-js.mdx:15]: next

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  endpoint: '<your-logto-endpoint>', // E.g. http://localhost:3001
+  baseUrl: '<your-nextjs-app-base-url>', // E.g. http://localhost:3000
+  cookieSecret: 'complex_password_at_least_32_characters_long',
+  cookieSecure: process.env.NODE_ENV === 'production',
+});
+```
+
+<AssumingUrl />
+
+## Configure sign in redirect URI
```
Comment (demonzoo) [inline docs/docs/recipes/integrate-logto/next-js.mdx:66]: sign-in

File: docs/docs/recipes/integrate-logto/next-js.mdx (hunk 1 above)
Comment (wangsijie) [reply docs/docs/recipes/integrate-logto/next-js.mdx]: updated

File: docs/docs/recipes/integrate-logto/next-js.mdx (hunk 2 above)
Comment (wangsijie) [reply docs/docs/recipes/integrate-logto/next-js.mdx]: updated

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  endpoint: '<your-logto-endpoint>', // E.g. http://localhost:3001
+  baseUrl: '<your-nextjs-app-base-url>', // E.g. http://localhost:3000
+  cookieSecret: 'complex_password_at_least_32_characters_long',
+  cookieSecure: process.env.NODE_ENV === 'production',
+});
+```
+
+<AssumingUrl />
+
+## Configure sign-in redirect URI
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/next-js.mdx:66]:
```suggestion
## Configure sign-in Redirect URI
```

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+<Link href="/api/logto/sign-in">
+  <a>Sign In</a>
+</Link>
+```
+
+:::note
+
+Before redirecting users to sign in route, make sure you have correctly configured Redirect URI in
+Admin Console.
+:::note
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/next-js.mdx:107]:
```suggestion
:::note

Before redirecting users to sign in route, make sure you have correctly configured Redirect URI in
Admin Console.

:::note
```

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+  return {
+    props: { user },
+  };
+});
+```
+
+## Protect API and pages
+
+### Protect API routes.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/next-js.mdx:158]:
```suggestion
### Protect API routes
```

File: docs/docs/recipes/integrate-logto/next-js.mdx [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    return;
+  }
+
+  response.json({
+    data: 'this_is_protected_resource',
+  });
+});
+```
+
+### Protect pages.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/next-js.mdx:178]:
```suggestion
### Protect pages
```

File: docs/docs/recipes/integrate-logto/next-js.mdx (hunk 3 above)
Comment (wangsijie) [reply docs/docs/recipes/integrate-logto/next-js.mdx:66]: In this article, title is treated as a sentence

**Review by gao-sun — APPROVED** [review]

**Pull request merged**